Makefile

```diff
@@ -164,3 +165,67 @@ ifneq ($(HAS_WEBAPP),)
 	rm -fr webapp/node_modules
 endif
 	rm -fr build/bin/
+
+# debug builds and deploys a debug version of the webapp, for only my architecture.
+.PHONY: debug
+debug: stop
+	./build/bin/manifest apply
+ifneq ($(HAS_SERVER),)
+	mkdir -p server/dist
+	cd server && env GOOS=darwin GOARCH=amd64 $(GO) build -gcflags "all=-N -l" -o dist/plugin-darwin-amd64
+
+	# uncomment for your own architecture. This speeds up the cycle.
+	#cd server && env GOOS=linux GOARCH=amd64 $(GO) build -gcflags "all=-N -l" -o dist/plugin-linux-amd64;
+	#cd server && env GOOS=windows GOARCH=amd64 $(GO) build -gcflags "all=-N -l" -o dist/plugin-windows-amd64.exe;
+	cd server && cp -r templates dist/templates
+endif
+	rm -rf dist/
+	mkdir -p dist/$(PLUGIN_ID)
+	cp $(MANIFEST_FILE) dist/$(PLUGIN_ID)/
+ifneq ($(HAS_SERVER),)
+	mkdir -p dist/$(PLUGIN_ID)/server/dist
+	cp -r server/dist/* dist/$(PLUGIN_ID)/server/dist/
+endif
+
+	mkdir -p ../mattermost-server/plugins
+	cp -r dist/* ../mattermost-server/plugins/
+
+	# if there is a current plugin running, kill it
```

Comment:
This is a workaround for a bug, as opposed to a standard operating procedure. Put a comment, with a ticket reference, so we remove these lines later?

Reply:
I fixed it so we don't need to do this. When disabling/reenabling the plugin, the process is killed (as long as there isn't a delve process attached -- which it checks for now). The original bug hasn't been fixed, but fixing it won't affect this now.